src/app/machines/components/MachineHeaderForms/MachineActionFormWrapper/TagForm/AddTagForm/AddTagForm.test.tsx

```diff
@@ -128,3 +166,68 @@ it("generates a deployed message for multiple machines", async () => {
       .startsWith("2 selected machines are deployed")
   ).toBe(true);
 });
+
+it("fetches deployed machine count for selected machines", async () => {
+  const store = mockStore(state);
+  const selectedMachines = { items: ["abc", "def"] };
+  render(
+    <Provider store={store}>
+      <MemoryRouter initialEntries={[{ pathname: "/tags", key: "testKey" }]}>
+        <AddTagForm
+          name="new-tag"
+          onTagCreated={jest.fn()}
+          selectedMachines={selectedMachines}
+        />
+      </MemoryRouter>
+    </Provider>
+  );
```

Comment:
```suggestion
  renderWithBrowserRouter(
    <AddTagForm
      name="new-tag"
      onTagCreated={jest.fn()}
      selectedMachines={selectedMachines}
    />,
    { route: "/tags", wrapperProps: { store } }
  );
```

Just make sure to change line 30 to the following to ensure this works:

```ts
const mockStore = configureStore<RootState, {}>();
```

And also import `renderWithBrowserRouter` from testing utils

Reply:
Sure, good suggestion!